apps/opik-backend/src/test/java/com/comet/opik/api/resources/v1/priv/DatasetsResourceTest.java

```diff
@@ -3773,6 +3791,16 @@ private static Set<Column> addDeprecatedFields(List<Map<String, JsonNode>> data)
         return columns;
     }
 
+    private String getType(Map.Entry<String, JsonNode> entry) {
+        return switch (entry.getValue().getNodeType()) {
+            case NUMBER -> {
+                var number = (NumericNode) entry.getValue();
+                yield number.isIntegralNumber() ? "Int64" : "Float64";
+            }
+            default -> StringUtils.capitalize(entry.getValue().getNodeType().name().toLowerCase());
+        };
+    }
```

Comment:
I'm not aware about the latest discussion with @andriidudar but I strongly recommend not leaking externally the ClicHouse types. Instead, I advice to map them to JSON types. It'd save us headaches like just having `numeric` instead of all these `Int64`, `Float64` etc.

This can be done in a separated PR prior to this.

Reply:
We can implement our own types; however, keep in mind that those are not DB types but JSON types, the JSON interpretation of the properties. In the end, the Front End will only use them to format the data. I will talk to @andriidudar about it